ecl/eclagent/eclagent.cpp

```diff
@@ -2001,8 +2001,8 @@ void EclAgent::doProcess()
         {
             if (w->getDebugValueBool("analyzeWorkunit", agentTopology->getPropBool("@analyzeWorkunit", true)))
             {
-                WuAnalyseOptions options;  // TODO: read options from configuration file
-                analyseWorkunit(w.get(), options);
+                IPropertyTree *analyzerOptions = agentTopology->queryPropTree("AnalyzerOptions");
```

Comment:
Any new config file definition should have a lower case leading for compatibility with new config.